eagle-stores/src/pages/Cart/components/CartCard.jsx

```diff
@@ -0,0 +1,106 @@
+import { useAuth, useData } from '../../../context';
+import { DeleteCart, IncDecCart } from '../../../services/Services';
+import { ACTION_TYPE } from '../../../utils/actionType';
+
+export const CartCard = ({ item }) => {
+  const { img, name, price, qty, _id, id } = item;
+
+  const { token } = useAuth();
+  const { dispatch } = useData();
+
+  const DeleteCartHandler = async () => {
+    try {
+      const res = await DeleteCart({ productId: _id, encodedToken: token });
+      if (res.status === 200 || res.status === 201) {
+        dispatch({
+          type: ACTION_TYPE.SETCART_LIST,
+          payload: { cartlist: res.data.cart },
+        });
+      }
+    } catch (err) {
+      console.log(err);
```

Comment:
You can remove this console log

Reply:
This console log is for error handling for now so that's why I used here console log